hoodie-client/src/test/java/com/uber/hoodie/TestHoodieClientBase.java

```diff
@@ -94,6 +99,33 @@ public void init() throws IOException {
     dataGen = new HoodieTestDataGenerator();
   }
 
+  @After
+  /**
+   * Properly release resources at end of each test
+   */
+  public void teardown() throws IOException {
```

Comment:
are subclasses calling super.teardown() failthfully?

Reply:
Currently, there is no teardown (@After)  methods in subclasses.

Reply:
understood.. just want to ensure this is called, since everything is reflection based.. Looks like it is.. Anyway, I suggest adding a teardown override in subclasses anyway.. since if someone does that down the line without calling super.teardown(), we are back to square one.

Reply:
Done.